synthtool/gcp/gapic_bazel.py

```diff
@@ -182,6 +189,10 @@ def _generate_code(
         cwd = os.getcwd()
         os.chdir(str(api_definitions_repo))
 
+        if clean_build:
```

Comment:
I'd love to refactor it `_generate_code()`, turn it inside out, so there are 7 top-level functions calling common functions as needed, with no if statements.  But, I'm afraid of breaking everything.

Sorry you had to deal with this, Mira.